dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"><title>Stars</title><link rel="stylesheet" href="style.7ab1061e.css"></head><body> <div class="stars--0 stars"> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> </div> <div class="stars--1 stars"> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> </div> <div class="stars--2 stars"> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> </div> <div class="stars--3 stars"> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> </div> <div class="stars--4 stars"> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> </div> <div class="stars--5 stars"> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> <span class="stars__star"></span> </div> </body></html>
```

Comment:
Better add only that files where you've written code or Readme, don't add any additional files to PR